lib/core/board.rb

```diff
@@ -5,27 +5,82 @@ class Board
   attr_reader :capacity
   attr_reader :size
 
-  def initialize (board_size)
+  def initialize(board_size)
     @size = board_size
     @capacity = size*size
     @grid = Array.new(capacity) {Cell.new()}
   end
 
-  def mark (position, symbol)
+  def mark(position, symbol)
    @grid[position].content = symbol
-   @grid
   end
 
   def is_full?
     @grid.all? { |cell| !cell.is_empty? }
   end
 
-  def get_content position
+  def get_content(position)
     @grid[position].content
   end
 
-  def invalid_cell position
+  def invalid_cell(position)
     !@grid[position].is_empty?
   end
-end
 
+  def has_a_winner?(players)
+    for player in players do
+      return true if is_a_winner?(player)
+    end
+    false
```

Comment:
This method can probably be simplified by the use of `select`: https://ruby-doc.org/core-2.2.0/Array.html#method-i-select